docs/UserGuide.md

```diff
@@ -11,6 +11,30 @@ transferable to other devices too!
 
 ## Features & Usage
 ### Feature: Candidate Management
+
+<u>**Add a candidate: `add c`**</u>
+
+Adds a candidate to the list of candidates.
+
+<u>Format:</u>
+
+    add c name=<NAME> email=<EMAIL> phone=<PHONE_NUMBER>
+
+<u>Example:</u><br/><br/>
+`add c name=<Bryan Seah> email=<bsah@gmail.com> phone=<12345678>` adds a candidate named Bryan Seah 
```

Comment:
The linebreak here seems a little out of place. Perhaps linebreak before "adds a ..."?

Reply:
Noted, made edits accordingly.